widgets/data_editor.py

```diff
@@ -870,6 +878,8 @@ def setup_widgets(self):
         self.unk_2f = self.add_integer_input("Unknown 0x2F", "unk_2f",
                                              MIN_UNSIGNED_BYTE, MAX_UNSIGNED_BYTE)
 
+        self.objdatalabel = self.add_button_input(
+            "Object-Specific Settings", "Fill With Default Values", self.fill_default_values)
```

Comment:
A shorter alternative could be **Reset to Default**. At the moment the label is a bit truncated, unless I make the splitter ridiculously wide.

(...although with the "expanding" size policy, as mentioned in the other comment, that would be a lesser problem.)